<?xml version="1.0"?>

<launch>

  <arg name="camera_info"  default="/thorvald_001/kinect2_front_camera/hd/camera_info"/>

  <!-- Start grape 2D annotation -->
  <node pkg="ros_assignment" type="grape.py" name="grape_2d_segmentation" output="screen">
    <param name="camera_info" value="$(arg camera_info)"/>
  </node>

</launch>
